assets/script.js

```diff
@@ -72,6 +72,28 @@ function getTrackInfo(userInput) {
       //show the result page after finish call
       $("#track-results-page").show(400);
     }
+    //"reading" the value of <li>
+    // $('#tracks').each(function() {
+      // $('#track-search-result>ol>li').on('click', function() {
+        $('.track-result').on('click', function() {
+          var getLyricsFor = $(this).text();
```

Comment:
Change this variable name to be consistent with the rest of the script e.g. track